lib/bot.js

```diff
@@ -119,15 +119,20 @@ Bot.prototype.run = function () {
     }
   });
   var prjNames = "";
-  _.each(self.config.projects, function (prj, index, list) {
-    pattern += prj;
-    prjNames += prj;
-    if (index != list.length - 1) {
-      pattern += "|";
-      prjNames += "|";
-    }
-  });
-  pattern += ")-[1-9][0-9]*)(\\+)?(?:(?=\\W)|$)";
+  if (self.config.projects !== undefined && self.config.projects.length > 0) {
+    pattern = "(?:\\W)((";
```

Comment:
See #38 comment on the old side.

Reply:
I suppose one of these days, I should write unit tests for all of this so it's more clear all the edge cases.  :-/